packages/py-api/py_api/controllers/teams_controller.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from bson.errors import InvalidId
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import t_col
```

Comment:
Why is this collection named `normal_teams` ?

Didn't we agree that there will be only one collection for teams? @IvanObreshkov

Reply:
Input? @IvanObreshkov @Aleks1908

Reply:
I will change the name. We are going to work only with one collection.